manifests/KDE/KDEConnect/1.4-449.yaml

```diff
@@ -0,0 +1,17 @@
+Id: KDE.KDEConnect
+Version: 1.4-449
+Name: KDE Connect
+Publisher: KDE
+License: GPL-2.0
+LicenseUrl: https://github.com/KDE/kdeconnect-kde/blob/master/LICENSES/GPL-2.0-or-later.txt
+Description: Enabling communication between all your devices. Made for people like you.
+Homepage: https://kdeconnect.kde.org/
+Tags: connect,phone,mobile,android,notification
+Installers:
+  - Arch: x64
+    Url: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/lastSuccessfulBuild/artifact/kdeconnect-kde-1.4-449-windows-msvc2019_64-cl-sideload.appx
```

Comment:
```suggestion
    Url: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/449/artifact/kdeconnect-kde-1.4-449-windows-msvc2019_64-cl-sideload.appx
```

Reply:
a new installer (version 1.4-450) was released an hour ago, so using `lastSuccessfulBuild` will always pick up the latest installer  every day regardless of the version listed in the manifest.

Reply:
Ok, thanks. Fixed it and re-validated the manifest!